jcloud/flow.py

```diff
@@ -192,7 +192,13 @@ async def status(self) -> Dict:
                     url=f'{WOLF_API}/{self.flow_id}', headers=self.auth_header
                 ) as response:
                     response.raise_for_status()
-                    return await response.json()
+                    _results = await response.json()
+                    # Extra handle for 'dashboards' key; the value is a map
+                    # but since we only have monitoring dashboard at the moment,
+                    # for simplicity we turn the map to a string for display purpose.
+                    if _results['dashboards']:
```

Comment:
Probably this is better here? 
```suggestion
                    if _results.get('dashboards', None) is not None:
```

Reply:
Discussed offline.